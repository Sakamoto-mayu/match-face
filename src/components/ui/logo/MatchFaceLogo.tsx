// Todo: 正式なロゴに差し替え
const MatchFaceLogo = () => {
    return <div>ロゴ</div>
}

export default MatchFaceLogo;
