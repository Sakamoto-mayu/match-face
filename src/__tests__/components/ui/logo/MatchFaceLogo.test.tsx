import MatchFaceLogo from "@/components/ui/logo/MatchFaceLogo";
import { render } from "@testing-library/react";

describe("ui/logo/MatchFacelogo.tsx", () => {
  it("スナップショット", () => {
    const { container } = render(<MatchFaceLogo />);
    expect(container).toMatchSnapshot();
  });
});
