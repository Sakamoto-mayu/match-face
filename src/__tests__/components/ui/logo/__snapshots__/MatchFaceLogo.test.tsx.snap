// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ui/logo/MatchFacelogo.tsx スナップショット 1`] = `
<div>
  <div>
    ロゴ
  </div>
</div>
`;
